src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -0,0 +1,84 @@
+package net.helpscout.api.utils;
+
+import lombok.SneakyThrows;
+
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @Author: ivan
+ * Date: 28.04.16
+ * Time: 0:29
+ */
+public class ParamsUtils {
+
+    @SneakyThrows
+    public static String setParams(String url, Map<String, String> params) {
```

Comment:
@bob983 I suppose Common Http Client is a good variant for lib with URI builder. Using it actually it is possible to remove custom Get,Post,Delete and Put implementation. But I'd like to make it as separate change. What do you think?